src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
We also need to add @Target annotation 
![image](https://github.com/mate-academy/jv-dependency-injection/assets/73792778/95e24e35-b238-4bdf-ac84-86f9cf876254)